services/QuillLMS/lib/tasks/migrate_incorrect_sequences_and_focus_points.thor

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative '../../config/environment'
+
+class MigrateIncorrectSequencesAndFocusPoints < Thor
```

Comment:
I think we generally prefer Rake to Thor, though I'm not sure if there's a rationale for this or if it's just convention. I think it might be worth considering using Rake here just to not have this outlier.

Reply:
@anathomical told me we prefer `thor`, haha! I'll put it as a topic in the engineering meeting.